### ISSUE TITLE
[CheckBox][High Contrast][Firefox] Disabled checkbox are not differentiated from enabled ones

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#2081]
### Bug Report

- __Browser__: Firefox

#### Describe the issue:

In Firefox + system high contrast mode, the checkbox component shows in same color for enabled as well as disabled case. There is no way to differentiate between the two.

#### Actual behavior:
Black on white mode: both enabled and disabled checkbox show up as white. In Edge, the diabled ones come as green.

#### Expected behavior:
Firefox should also follow high contrast mode and show disabled checkbox in green

Related issue: https://github.com/OfficeDev/office-ui-fabric-react/issues/1946

Screenshot:
![image](https://user-images.githubusercontent.com/11937034/27630995-cc427900-5c14-11e7-8301-282a2d5b39e2.png)

Here first one is disabled but second one is not.

**Comment by prmittal** [comment]
Behavior on Edge:

![image](https://user-images.githubusercontent.com/11937034/27631109-12db0dc8-5c15-11e7-819f-c966a84a0635.png)
